Documentation/openshift-issues.md

```diff
@@ -0,0 +1,50 @@
+---
+title: OpenShift Common Issues
+weight: 10620
+indent: true
+---
+
+# OpenShift Common Issues
+
+## Enable Monitoring in the Storage Dashboard
+
+OpenShift Console uses OpenShift Prometheus for monitoring and populating data in Storage Dashboard. Additional configuration is required to monitor the Ceph Cluster from the storage dashboard.
+
+1. Enable Ceph Cluster monitoring
+
+    Follow [ceph-monitoring/prometheus-alerts](https://github.com/rook/rook/blob/master/Documentation/ceph-monitoring.md#prometheus-alerts).
+
+2. Set label on namespace
```

Comment:
suggestion: "Set the required label on the namespace"

Reply:
ack